prow/commands.md

```diff
@@ -4,11 +4,13 @@
 
 Command | Plugin | Who can run it | Description
 --- | --- | --- | --- | ---
+`/assign [@userA @userB @etc]` | [assign](./plugins/assign) | org members | Assigns specified people (or yourself if no one is specified)
```

Comment:
I wasn't going for alphabetized commands. Probably we should just put them together.